ansible/roles/metrika-agent-install/templates/agent.yml.j2

```diff
@@ -0,0 +1,92 @@
+---
+platform:
+
+  # api_key: authentication key for the Metrika platform
+  api_key: {{ metrika_agent_api_key }}
+
+  # batch_n: integer, the maximum number of metrics/events to be emitted at
+  # each publish operation to the platform.
+  batch_n: 1000
+
+  # max_publish_interval: duration, the maximum interval to wait between two
+  # consecutive publish operations to the platform.
+  max_publish_interval: 5s
+
+  # transport_timeout: duration, configures the timeout for publish
+  # operations to the platform.
+  transport_timeout: 10s
+
+  # addr: string, network address of the platform for publishing
+  addr: agent.sandboxes.aws.metrika.co:443
+
+  # uri: string, platform publishing endpoint
+  uri: /
+
+buffer:
+
+  # max_heap_alloc: integer, the maximum bytes of allocated heap objects as reported
+  # by Go's runtime memstats, that are allowed by the agent before it starts
+  # dropping metrics (default: 50MB).
+  max_heap_alloc: 52428800
+
+  # ttl: duration, a Time-To-Live for any buffered data. Data will never expire
+  # for zero TTL values and stay in the buffer until they are evicted for publishing.
+  ttl: 0s
+
+runtime:
+
+  logging:
+
+    # outputs: list of outputs for runtime logs.
+    #
+    # Possible values are one of the standard I/O streams # (i.e stdout, stderr)
+    # or a full/relative path to a filename (i.e logs/agent.log).
+    outputs:
+      - stdout
+
+    # level: logging level for runtime logs.
+    #
+    # Possible values are: info, warning (recommended), debug, error.
+    level: warning
+
+    # disable_fingerprint_validation: disables fingerprint validation on startup.
+    #
+    # Fingerprint validation is enabled by default and the agent will exit
+    # immediately if checksums of the newly retrieved hostname and the cached do
+    # not match. Checksum is cached under /Users/sjonpaul.brown/.cache/metrikad/fingerprint.
+    disable_fingerprint_validation: false
+
+  # metrics_addr: string, network address to get Prometheus metrics about the
+  # agent's runtime.
+  metrics_addr: :9000
```

Comment:
This variable should not be hardcoded, but rather configurable by the end-user.

The default value should be: `127.0.0.1:9000`

The reason is that the current hardcoded value of `:9000` starts the /metrics HTTP server of the agent listening on 
`0.0.0.0:9000`.

Reply:
Addressed in a2516967.